source/include/dqm4hep/MonitorElement.h

```diff
@@ -266,6 +266,14 @@ namespace dqm4hep {
       /** Convert the scalar value to std::string
        */
       std::string ToString() const;
+      
+      /**
+       *  @brief  Whether the scalar is equal to an other one.
+       *          Compare only the value itself
+       *          
+       *  @param  other the other scalar object
+       */
+      Bool_t IsEqual(const TObject *other) const override;
```

Comment:
Why not declare other as a TScalarObject directly? This would prevent trying to compare any other TObject

Reply:
This method is a virtual method from TObject. I can't change the prototype !

Reply:
Didn't check what it was overriding. Good point